### PR TITLE
Editor Setup

### PR DESCRIPTION
**Pull request by Jam-Hejrati**
translate editor-setup to persian

### CODE REVIEW

**Comment by rezvani2647** [comment]
Greate. Thank you for your time and effort.
but there is a slitly change you need to apply before we review you PR.
Please read [Style Guid](https://github.com/reactjs/fa.react.dev/blob/main/STYLE-GUIDE.md) and make sure your translation meets the style guid standards. (for example remove all `div` tags).

Thank you.

**Comment by Jam-Hejrati** [comment]
Thanks for guidance.
I attempted for fixing issues according to (Style Guide) [https://github.com/reactjs/fa.react.dev/blob/main/STYLE-GUIDE.md]

**Comment by Jam-Hejrati** [comment]
Thanks for guidance
I attempted to fix and delete redundant tags according to (Style Guide) [https://github.com/reactjs/fa.react.dev/blob/main/STYLE-GUIDE.md]

**Comment by Jam-Hejrati** [comment]
Thanks for guidance
I attempted to fix and delete redundant tags according to Style Guide

**Comment by Jam-Hejrati** [comment]
Hi @rezvani2647 
Why did the deployment check fail?
I can't see any detail of checks for failed reason to fix it.

File: src/content/learn/editor-setup.md [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
+* [WebStorm](https://www.jetbrains.com/webstorm/) یک محیط توسعه‌ی یکپارچه که مخصوص جاوااسکریپت طراحی شده است.
+* [Sublime Text](https://www.sublimetext.com/) از JSX و تایپ‌اسکریپت پشتیبانی میکند، دارای [برجسته‌ساز syntax](https://stackoverflow.com/a/70960574/458193) و تکمیل خودکار داخلی است.
+* [Vim](https://www.vim.org/) یک ویرایشگر متن با قابلیت تنظیم بالا است که به منظور ایجاد و تغییر هر نوع متن، بسیار کارآمد ساخته شده است. در اکثر سیستم‌های یونیکس و با Apple OS X به‌عنوان «vi» گنجانده شده‌است.
 
-## Recommended text editor features {/*recommended-text-editor-features*/}
+## قابلیت های پیشنهادی برای ویرایشگر متن {/*recommended-text-editor-features*/}
 
-Some editors come with these features built in, but others might require adding an extension. Check to see what support your editor of choice provides to be sure!
+بعضی ویرایشگرها به صورت داخلی این قابلیت ها را دارند، اما بعضی دیگر نیاز به اضافه کردن افزونه هستند. بررسی کنید تا ببینید ویرایشگر انتخابی شما چه پشتیبانی ارائه می‌دهد تا مطمئن شوید!
```
Comment (faeghe-hajiabadi) [inline src/content/learn/editor-setup.md:30]: املای مطمئن درست نوشته نشده است.

**Review by faeghe-hajiabadi — CHANGES_REQUESTED** [review]

**Review by faeghe-hajiabadi — APPROVED** [review]

**Review by rezvani2647 — APPROVED** [review]

**Pull request merged**